examples/k8s/k8s-helloworld-25p-v2.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 IBM Corporation
```

Comment:
and please add a note about customizing the tests for kubernetes, in  testing/test-scripts/helloworld.sh above the following code snippet (75/25 case)
```bash
 echo "Set/verify the Hello World route to 75%/v1 25%/v2"
    echo "  Invoking a8ctl to set a rule to split the traffic."
    $A8CLI rule-create -f $SCRIPTDIR/helloworld-v1-v2-route-rules.json > /tmp/a8ctl_output.txt
```